frontend/src/components/CostTable.vue

```diff
@@ -131,21 +132,13 @@ const columns: any = computed(() => [
       compare: (row1: DataRow, row2: DataRow) => {
         const a = row1.region.toLocaleLowerCase();
         const b = row2.region.toLocaleLowerCase();
-        const len = a.length > b.length ? b.length : a.length;
-        for (let i = 0; i < len; i++) {
-          if (a[i] < b[i]) {
-            return false;
-          } else if (a[i] > b[i]) {
-            return true;
-          }
+        const stringComp = a.localeCompare(b);
+        if (stringComp !== 0) {
+          return stringComp;
         }
 
-        // if tow region are identical, sort them with id
-        if (a.length == b.length) {
-          return row1.id - row2.id;
-        }
-
-        return a.length - b.length;
+        // if tow region are identical, sort them with instance's id
```

Comment:
tow -> two. Use full English sentence for comments, end with period.

// If two regions are identical, sort them with instance id.